lib/btc/BtcRpc.js

```diff
@@ -45,10 +45,33 @@ class BtcRpc {
     return this.asyncCall('cmdlineUnlock', [time]);
   }
 
+  async sendMany({ obj, options }) {
+    for (let tx in obj) {
+      obj[tx] /= 1e8;
+    }
+    let array = [];
+    array.push('');
+    array.push(obj);
+    if (options) {
+      array.push(options);
+    }
+    return this.asyncCall('sendMany', array);
+  }
+
   async sendToAddress({ address, amount }) {
     return this.asyncCall('sendToAddress', [address, amount / 1e8]);
   }
 
+  async unlockAndSendManyBatched({ obj, passphrase, options }) {
+    if (passphrase === undefined) {
+      passphrase = await passwordPromptAsync('> ');
+    }
+    await this.walletUnlock({ passphrase, time: 10 });
```

Comment:
10 is too low for many wallets

Reply:
What time do you suggest? I was following `unlockAndSendToAddress` as an example

Reply:
That one is likely broken as well.

Reply:
in `unlockAndSendToAddressMany` time is defaulted to 10800, I have done this in my method and also `unlockAndSendToAddress`, my guess is this is what John wanted. Will confirm when he gets back

Reply:
John confirmed, 10800 seconds is fine (3 hours)